doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

```diff
@@ -0,0 +1,78 @@
+
+================================
+CORTX on Open Virtual Appliance
+================================
+An Open Virtual Appliance (OVA) is a Virtual Machine (VM) image that consists of a pre-installed and pre-configured operating system as well as one or more applications packaged for easy deployment and testing.  This document describes how to use a CORTX OVA for the purposes of single-node CORTX testing. 
+For our Japanese community, this document has been translated and is available `here <https://qiita.com/Taroi_Japanista/items/0ac03f55dce3f7433adf>`_.
+
+***********************
+Recommended Hypervisors
+***********************
+All of the following hypervisors should work: `VMware ESX Server <https://www.vmware.com/products/esxi-and-esx.html>`_,
+`VMware vSphere <https://www.vmware.com/products/vsphere.html>`_,
+`VMware Workstation <https://www.vmware.com/products/workstation-pro.html>`_,
+`VMware Fusion <https://www.vmware.com/in/products/fusion/fusion-evaluation.html>`_
+
+**Important**: If you are running the VM in any of the VMWare hypervisors, it is not recommended to use VMware Tools, as CORTX may break due to kernel dependencies. For the same reason, please do not update the operating system in the image as that also might cause it to fail.
+
+**Specifications:**
+
+- CORTX OVA image is created with following minimum configuration:
+
+  - RAM: 8GB
```

Comment:
Update the spec as per new OVA as,

- RAM: 10GB
- Processor: 8

Reply:
Done.